tests/perf-regression-2mv.yaml

```diff
@@ -17,7 +17,7 @@ experimental: 'true'
 
 # email definitions to send regression report
 send_email: true
-email_recipients: ['roy@scylladb.com', 'larisau@scylladb.com', 'julia.yakovlev@scylladb.com']
+email_recipients: ['roy@scylladb.com', 'larisau@scylladb.com', 'julia.yakovlev@scylladb.com', 'qa@scylladb.com']
```

Comment:
roy, larisa, yulia are all in the qa mail list. So I think you can remove other emails and leave only qa@scylladb.com